scene/gui/option_button.cpp

```diff
@@ -329,11 +359,10 @@ void OptionButton::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("get_popup"), &OptionButton::get_popup);
 
-	ClassDB::bind_method(D_METHOD("_set_items"), &OptionButton::_set_items);
-	ClassDB::bind_method(D_METHOD("_get_items"), &OptionButton::_get_items);
-
-	ADD_PROPERTY(PropertyInfo(Variant::ARRAY, "items", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_NO_EDITOR | PROPERTY_USAGE_INTERNAL), "_set_items", "_get_items");
-	// "selected" property must come after "items", otherwise GH-10213 occurs.
+	ClassDB::bind_method(D_METHOD("set_item_count"), &OptionButton::set_item_count);
+	ClassDB::bind_method(D_METHOD("get_item_count"), &OptionButton::get_item_count);
+	// "selected" property must come after "items_count", otherwise GH-10213 occurs.
+	ADD_ARRAY_COUNT("Items", "items_count", "set_item_count", "get_item_count", "popup/item_");
```

Comment:
```suggestion
	// "selected" property must come after "item_count", otherwise GH-10213 occurs.
	ADD_ARRAY_COUNT("Items", "item_count", "set_item_count", "get_item_count", "popup/item_");
```

See #55685.